internal/auth/manager.go

```diff
@@ -269,13 +265,26 @@ func (m *Manager) authenticateJWT(req *Request) error {
 		return fmt.Errorf("JWT not provided")
 	}
 
-	var cc customClaims
-	_, err = jwt.ParseWithClaims(v["jwt"][0], &cc, keyfunc)
+	token, err := jwt.Parse(v["jwt"][0], keyfunc)
 	if err != nil {
 		return err
 	}
 
-	if !matchesPermission(cc.MediaMTXPermissions, req) {
+	claims := token.Claims.(jwt.MapClaims)[m.JWTClaimKey].([]interface{})
```

Comment:
this is prone to panics since `token.Claims.(jwt.MapClaims)[m.JWTClaimKey]` can be `nil`